pkg/commands/product/bot_management/disable.go

```diff
@@ -0,0 +1,77 @@
+package bot_management
+
+import (
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+	"github.com/fastly/go-fastly/v9/fastly/products/bot_management"
+
+	"github.com/fastly/cli/pkg/api"
+	"github.com/fastly/cli/pkg/argparser"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// DisableFn is a dependency-injection point for unit tests to provide
+// a mock implementation of the API operation.
+var DisableFn = func(client api.Interface, serviceID string) error {
```

Comment:
Probably a Go idiom or just a style preference of my own from familiarity with the standard library, but i suggest `DisableFunc` and friends.